.github/workflows/iroha2-dev-pr.yml

```diff
@@ -19,6 +19,11 @@ jobs:
         with:
           path: ${{ env.CACHE_PATHS }}
           key: ${{ hashFiles('Cargo.toml') }}
+      - name: Print all info
+        run: |
+          cargo version
+          lscpu
+          free -h
```

Comment:
Do we still need all the info? Maybe only `cargo version` now.

Reply:
Yeah, I think it is good to keep it for record. Have seen it in several other projects